src/com/example/ppj05/Task03.java

```diff
@@ -0,0 +1,53 @@
+package com.example.ppj05;
+
+    /*
+        Write a program which reads three integers (say, a, b and c), then prints these three
+        numbers
+        System.out.println(a + " " + b + " " + c);
+        and then rearranges the values in these variables in such a way that a contains the
+        smallest of the three numbers, b — the middle one, and c — the largest.
+        Print again System.out.println(a + " " + b + " " + c);
+        and you shoud see the same three numbers, but in ascending order.
+        Any two (or all three) numbers may be equal. Do not use arrays or Strings!
+    */
+import java.util.Scanner;
+
+    public class Task03 {
+
+        public static void main(String[] args) {
+            Scanner scanner = new Scanner(System.in);
+
+            int a = scanner.nextInt();
+            int b = scanner.nextInt();
+            int c = scanner.nextInt();
+
+            System.out.println(a + " " + b + " " + c);
+
+            int min, mid, max;
+
+            min = Math.min(Math.min(a, b), c);
```

Comment:
We should not use Math utills methods in this task. This task needed for practice if else constructions